db/schema.rb

```diff
@@ -55,6 +55,30 @@
   add_index "identities", ["session_uuid"], name: "index_identities_on_session_uuid", unique: true, using: :btree
   add_index "identities", ["user_id"], name: "index_identities_on_user_id", using: :btree
 
+  create_table "profiles", force: :cascade do |t|
+    t.integer  "user_id",                      null: false
+    t.boolean  "active",       default: false, null: false
+    t.datetime "verified_at"
+    t.datetime "activated_at"
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
+    t.string   "first_name"
+    t.string   "middle_name"
+    t.string   "last_name"
+    t.string   "address1"
+    t.string   "address2"
+    t.string   "city"
+    t.string   "state"
+    t.string   "zipcode"
+    t.string   "ssn"
+    t.string   "dob"
+    t.string   "phone"
+    t.string   "vendor"
+  end
+
+  add_index "profiles", ["user_id", "active"], name: "profiles_active_user_idx", unique: true, where: "(active = true)", using: :btree
```

Comment:
I had originally given it the name ending in `_idx` to indicate the constraint was an index. Since Rails can construct a partial index (with the `where` clause) I don't need to give it an explicit name anymore, and can change it to a `change` instead of `up/down`